src/lib.rs

```diff
@@ -293,15 +210,15 @@ pub struct Evaluator<'tcx> {
     /// Miri does not expose env vars from the host to the emulated program
     pub(crate) env_vars: HashMap<Vec<u8>, Pointer>,
 
-    /// Places that were suspended by the validation subsystem, and will be recovered later
-    pub(crate) suspended: HashMap<DynamicLifetime, Vec<ValidationQuery<'tcx>>>,
+    /// Use the lifetime
```

Comment:
lolwat? Are you going to need it again in the future? Or why not just remove it? Too much churn?

Reply:
You mean remove the lifetime? Hm, I didn't remember adding it so I just did not touch it.^^  But also indeed I am not sure yet which extra data I will add again for the Stacked Borrows stuff.